.github/workflows/release.yml

```diff
@@ -33,9 +33,22 @@ jobs:
       - name: Install Dependencies
         run: pnpm install --ignore-scripts
 
-      - name: Create Release Pull Request or Publish to npm
+      - name: Create Release Pull Request
         uses: changesets/action@v1
         with:
-          publish: pnpm release
+          version: pnpm changeset version
```

Comment:
`pnpm changeset tag` only creates the tags, but if you add it to the `changesets/action`, it will also push the tags to the remote.

Here is an example:
https://github.com/tedraykov/reaction/actions/runs/3202693539/jobs/5231938200#step:7:69